pcap-linux.c

```diff
@@ -1886,13 +1895,8 @@ get_if_flags(const char *name, bpf_u_int32 *flags, char *errbuf)
 			/*
 			 * OK, this OS version or driver doesn't support
 			 * asking for this information.
-			 * XXX - distinguish between "this doesn't
```

Comment:
What do `EOPNOTSUPP` and `EINVAL` indicate here? Does that make the distinction in question?